xmtp_mls/src/groups/mod.rs

```diff
@@ -942,6 +947,23 @@ impl MlsGroup {
             })
     }
 
+    /// Find the `consent_state` of the group
+    pub fn consent_state(&self) -> Result<ConsentState, GroupError> {
+        let conn = self.context.store.conn()?;
+        conn.find_group(self.group_id.clone())
+            .map_err(GroupError::from)
```

Comment:
```suggestion
            .map_err(GroupError::from)?
```



Then you shouldn't need the `and_then`